appcues/src/test/java/com/appcues/statemachine/states/BeginningExperienceStateTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.appcues.statemachine.states
+
+import com.appcues.data.model.Experience
+import com.appcues.statemachine.Action.StartExperience
+import com.appcues.statemachine.State
+import com.appcues.statemachine.effects.PresentationEffect
+import com.google.common.truth.Truth.assertThat
+import io.mockk.mockk
+import org.junit.Test
+
+internal class BeginningExperienceStateTest {
+
+    @Test
+    fun `BeginningExperienceState SHOULD be instance of STATE`() {
```

Comment:
checking to see if State is instance o State and what are the values on currentExperience and currentStepIndex (same test for each StateTest)